tests/integration/relations/pgbouncer_provider/application-charm/actions.yaml

```diff
@@ -25,3 +25,24 @@ run-sql:
   - query
   - relation-id
   - relation-name
+
+test-tls:
+  description: Checks if TLS is enabled on the backend database.
+  params:
+    dbname:
+      description: database on which to run the command
+      type: string
+    relation-id:
+      description: id of the relation under test
+      type: integer
+    relation-name:
+      description: name of the relation under test
+      type: string
+    readonly:
+      description: whether or not to use the readonly endpoint
+      type: boolean
+      default: false
+  required:
+  - dbname
+  - relation-id
+  - relation-name
```

Comment:
We can't simply query for `show ssl`, since it will report what is true for postgresql, not what is true for pgbouncer